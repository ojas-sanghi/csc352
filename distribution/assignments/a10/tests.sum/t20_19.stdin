90575180844786738559
29196571953659721041
96246057351221910564
91949881344980428184
81772131062696880770
21661175070424104369
95242813632546835296
89871195992717957762
65491488121993579075
4196164082998413151
20884103208932803905
65784328622597601689
92345393535505812378
30640684320450195675
42231778109638580685
23917638393854488413
99662122959804625674
55507276122747641985
91207316329255221067
49292904412125636193
