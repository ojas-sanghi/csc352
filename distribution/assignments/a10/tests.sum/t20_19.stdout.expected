90575180844786738559 (audit=1)
sum: 90575180844786738559 (audit=1)
29196571953659721041 (audit=1)
sum: 119771752798446459600 (audit=1)
96246057351221910564 (audit=1)
sum: 216017810149668370164 (audit=1)
91949881344980428184 (audit=1)
sum: 307967691494648798348 (audit=1)
81772131062696880770 (audit=1)
sum: 389739822557345679118 (audit=1)
21661175070424104369 (audit=1)
sum: 411400997627769783487 (audit=1)
95242813632546835296 (audit=1)
sum: 506643811260316618783 (audit=1)
89871195992717957762 (audit=1)
sum: 596515007253034576545 (audit=1)
65491488121993579075 (audit=1)
sum: 662006495375028155620 (audit=1)
4196164082998413151 (audit=1)
sum: 666202659458026568771 (audit=1)
20884103208932803905 (audit=1)
sum: 687086762666959372676 (audit=1)
65784328622597601689 (audit=1)
sum: 752871091289556974365 (audit=1)
92345393535505812378 (audit=1)
sum: 845216484825062786743 (audit=1)
30640684320450195675 (audit=1)
sum: 875857169145512982418 (audit=1)
42231778109638580685 (audit=1)
sum: 918088947255151563103 (audit=1)
23917638393854488413 (audit=1)
sum: 942006585649006051516 (audit=1)
99662122959804625674 (audit=1)
sum: 1041668708608810677190 (audit=1)
55507276122747641985 (audit=1)
sum: 1097175984731558319175 (audit=1)
91207316329255221067 (audit=1)
sum: 1188383301060813540242 (audit=1)
49292904412125636193 (audit=1)
sum: 1237676205472939176435 (audit=1)
